scripts/styles.css

```diff
@@ -77,6 +86,56 @@ input:-webkit-autofill:active {
 }
 
 h2 {
-  border-top: 2px solid black;
   padding-top: 20px;
 }
+
+#top-navigation {
```

Comment:
- [ ] [OPTIONAL] Add some font-size to text in this section. They are too small. Just an observation, though. The same suggestion goes to the footer.